nat-lab/tests/telio.py

```diff
@@ -27,6 +27,18 @@ class State(Enum):
     Connected = "connected"
 
 
+class LinkState(Enum):
+    Down = "down"
+    Up = "up"
+
+
+@dataclass
+class LinkStateEventsCount:
+    none: int = 0
+    up: int = 0
+    down: int = 0
```

Comment:
It's not bad as it is but we have deterministic results(given perfect network conditions) so I would prefer having an explicit ordering of events as an improvement(up, down, then up again) for example. Without order it might hide subtle bugs.